fe/src/main/java/org/apache/doris/catalog/FunctionSet.java

```diff
@@ -513,9 +515,9 @@ public void init() {
     public static final String BITMAP_UNION = "bitmap_union";
     public static final String BITMAP_UNION_INT = "bitmap_union_int";
     public static final String BITMAP_COUNT = "bitmap_count";
-    public static final String BITMAP_EMPTY = "bitmap_empty";
-    public static final String TO_BITMAP = "to_bitmap";
 
+    public static final Set<String> BITMAP_LOAD_FNS =
```

Comment:
better to make it a immutable tree set to avoid callers calling tolowercase every time

Reply:
OK